buildSrc/src/main/kotlin/Dependencies.kt

```diff
@@ -92,6 +95,11 @@ object Dependencies {
         const val playServicesCastFramework = "com.google.android.gms:play-services-cast-framework:${Versions.playServicesCast}"
     }
 
+    object Car {
```

Comment:
Shouldn't call this "car" since the media library / mediabrowser feature is not specifically for cars.